namer/namer.cc

```diff
@@ -1353,6 +1353,7 @@ class SymbolDefiner {
             }
         }
         sym = ctx.state.enterStaticFieldSymbol(ctx.locAt(staticField.lhsLoc), scope, name);
+        sym.data(ctx)->addLoc(ctx, ctx.locAt(staticField.lhsLoc));
```

Comment:
Unlike with `singletonClass` and `enterMethodSymbol`, I think that we likely have a much smaller scope for `enterStaticFieldSymbol`, and we could consider doing something like we did with `staticInitForFile` and `staticInitForClass`, where if the symbol exists, we call `addLoc`.

Reply:
Sounds good to me, let's try that and see what fails